src/main/kotlin/io/github/anki/anki/controller/dto/DeckDto.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.anki.anki.controller.dto
+
+import jakarta.validation.constraints.NotBlank
+
+data class NewDeckRequest(
+    @field:NotBlank(message = "should not be blank")
```

Comment:
[Look the comment about default message](https://github.com/OleksandrKurta/Anki/pull/6/files#r1713061099)